frontend/src/components/admin/ManageStories.tsx

```diff
@@ -15,13 +16,23 @@ const ManageStories = () => {
     },
   });
 
+  const history = useHistory();
+  const routeChange = () => {
+    history.push("/import-story");
```

Comment:
Used this function to change the route. Should I instead make the import story button into a link and use href?

Reply:
this is fine with me!